birdhouse/optional-components/all-public-access/all-public-access-magpie-permission.cfg

```diff
@@ -0,0 +1,107 @@
+permissions:
+  - service: catalog
+    permission: getcapabilities
+    group: anonymous
+    action: create
+
+  - service: catalog
+    permission: describeprocess
+    group: anonymous
+    action: create
+
+  - service: catalog
+    permission: execute
+    group: anonymous
+    action: create
+
+
+  - service: finch
+    permission: getcapabilities
+    group: anonymous
+    action: create
+
+  - service: finch
+    permission: describeprocess
+    group: anonymous
+    action: create
+
+  - service: finch
+    permission: execute
+    group: anonymous
+    action: create
+
+
+  - service: flyingpigeon
+    permission: getcapabilities
+    group: anonymous
+    action: create
+
+  - service: flyingpigeon
+    permission: describeprocess
+    group: anonymous
+    action: create
+
+  - service: flyingpigeon
+    permission: execute
+    group: anonymous
+    action: create
+
+
+  - service: hummingbird
+    permission: getcapabilities
+    group: anonymous
+    action: create
+
+  - service: hummingbird
+    permission: describeprocess
+    group: anonymous
+    action: create
+
+  - service: hummingbird
+    permission: execute
+    group: anonymous
+    action: create
+
+
+  - service: malleefowl
+    permission: getcapabilities
+    group: anonymous
+    action: create
+
+  - service: malleefowl
+    permission: describeprocess
+    group: anonymous
+    action: create
+
+  - service: malleefowl
+    permission: execute
+    group: anonymous
+    action: create
+
+
+  - service: raven
+    permission: getcapabilities
+    group: anonymous
+    action: create
+
+  - service: raven
+    permission: describeprocess
+    group: anonymous
+    action: create
+
+  - service: raven
+    permission: execute
+    group: anonymous
+    action: create
+
+
+  - service: thredds
+    permission: read
+    group: anonymous
+    action: create
+
+
+  - service: thredds
+    permission: read
+    user: authtest
```

Comment:
This is same as for group anonymous above, why need it? 

If to give specific read access to the secure folder, I think you'll have to add `resource: /birdhouse/testdata/secure` (untested syntax).